tensorboard/uploader/uploader.py

```diff
@@ -204,99 +186,79 @@ class _OutOfSpaceError(Exception):
 class _RequestBuilder(object):
```

Comment:
Oh, we should probably rename this `_BatchedRequestSender` or similar now.  WDYT?  `*EventWriter` would make sense locally but I don't want to collide with the other usage).  `*EventUploader`?

Reply:
Agreed. Sure, `_BatchedRequestSender` or similar sounds good to me.
Agreed that `EventWriter` would be confusing, and I think that
`EventUploader` would be confusing for the same reason.